### PR TITLE
[EKA-000]CM_APP_37:Invalid attempts on the entered consent pin

### PR DESCRIPTION
**Pull request by KrupaDesai22**

### CODE REVIEW

File: features/consent/src/test/java/in/projecteka/jataayu/consent/viewmodel/UserVerificationViewModelTest.kt [hunk 1]
```diff
@@ -98,6 +108,25 @@ class UserVerificationViewModelTest {
         Mockito.verify(call).enqueue(any())
     }
 
+    @Test
+    fun shouldReturnAccountLockWithExceedLimit(){
```
Comment (mahabaleshwarhnr) [inline features/consent/src/test/java/in/projecteka/jataayu/consent/viewmodel/UserVerificationViewModelTest.kt]: can you use back tick and write method like a sentence. so test easily readbale

File: features/consent/src/main/res/layout/user_verification_fragment.xml [hunk 2]
```diff
@@ -26,6 +26,24 @@
         android:layout_height="match_parent"
         tools:context=".ui.fragment.ConfirmPinFragment">
 
+        <TextView
+                android:id="@+id/lblBlockUser"
```
Comment (mahabaleshwarhnr) [inline features/consent/src/main/res/layout/user_verification_fragment.xml:30]: I think top padding required bit more. can you pls check figma?

Comment (KrupaDesai22) [reply features/consent/src/main/res/layout/user_verification_fragment.xml:30]: its correct!

File: features/consent/src/test/java/in/projecteka/jataayu/consent/viewmodel/UserVerificationViewModelTest.kt (hunk 1 above)
Comment (KrupaDesai22) [reply features/consent/src/test/java/in/projecteka/jataayu/consent/viewmodel/UserVerificationViewModelTest.kt]: updated

**Pull request merged**